internal/state/state.go

```diff
@@ -65,8 +65,8 @@ type EvolutionContractState interface {
 }
 
 type OwnershipBlockState interface {
```

Comment:
Rename these interfaces to `OwnershipSyncState` and `ÈvolutionSyncState` please

Reply:
changed